src/tests/ftest/control/dmg_system_reformat.py

```diff
@@ -91,8 +99,9 @@ def test_dmg_system_reformat(self):
                 self.get_dmg_command().result.stdout_text))
 
         # Create last pool now that memory has been wiped.
-        self.add_pool_qty(quantity=1, connect=False)
+        pools.append(add_pool(self, connect=False))
 
         # Lastly, verify that last created pool is in the list
         pool_uuids = self.get_dmg_command().get_pool_list_uuids()
-        self.assertEqual(pool_uuids[0].lower(), self.pool[-1].uuid.lower())
+        self.assertEqual(
+            pool_uuids[0].lower(), pools[-1].uuid.lower(), "{} missing from list".format(pools[-1]))
```

Comment:
`str(pools[-1])` won't show the uuid, right? If so
```suggestion
            pool_uuids[0].lower(), pools[-1].uuid.lower(), "{} missing from list".format(pools[-1]).uuid.lower())
```

Reply:
Yes it will https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/test_utils_pool.py#L181-L183